blog/Community/2015-03_11_Announce_of_the_mailing_lists.md

```diff
@@ -0,0 +1,16 @@
+Title: Announce of the mailing lists
+Date: 2015-03-11
+Comments_at: https://github.com/FreshRSS/freshrss.org/issues/21
+
+Did you know that FreshRSS has two mailing lists since few weeks?
+
+In order to welcome our users and developers, we have decided to open them so we can discuss about FreshRSS without forcing the use of GitHub and still keeping a community discussion.
```

Comment:
You can change `we have decided` by `we decided`. Both are correct but the later is less formal.